ui.py

```diff
@@ -168,7 +168,7 @@ def draw(self, context):
         else:
             uv_obj = get_object(context, context.object)
 
-        labeltext = "baking UV from " + uv_obj.name
+        labeltext = "Bake UVs of " + uv_obj.name
```

Comment:
should be from since it indicates the second selected object when baking from to

Reply:
👌